narwhals/expr.py

```diff
@@ -1765,6 +1765,111 @@ def gather_every(self: Self, n: int, offset: int = 0) -> Self:
             lambda plx: self._call(plx).gather_every(n=n, offset=offset)
         )
 
+    # need to allow numeric typing
+    # TODO @aivanoved: make type alias for numeric type
+    def clip(
+        self,
+        lower_bound: IntoExpr | Any | None = None,
+        upper_bound: IntoExpr | Any | None = None,
+    ) -> Self:
+        r"""
+        Clip values in the Series.
+        Arguments:
+            lower_bound: Lower bound value.
+            upper_bound: Upper bound value.
+        Examples:
```

Comment:
In this example we should exemplify the behavior on an expr, not on a series, namely something like `df.select(nw.col("a").clip(2))`

Reply:
thank you for the catch, I had forgotten to update after copy pasting